data/scripts.js

```diff
@@ -233,7 +234,8 @@ exports.BattleScripts = {
 				}
 			}
 			if (!move.ignoreEvasion) {
-				if (target.boosts.evasion > 0 && !move.ignorePositiveEvasion) {
+				var pokemonAbility = (pokemon.ignore['Ability'] === true || pokemon.ignore['Ability'] === 'A') ? {} : pokemon.getAbility();
+				if (target.boosts.evasion > 0 && !move.ignorePositiveEvasion && !pokemonAbility.ignoreEvasion) {
```

Comment:
Shouldn't `!pokemonAbility.ignoreEvasion` be with `!move.ignoreEvasion` two lines up, to be consistent with the accuracy checks? Otherwise Unaware won't ignore negative evasiveness like it should.